src/Platform/PlatformPosix.cpp

```diff
@@ -378,4 +335,29 @@ std::vector<std::string> CrossPlatform::getDataPaths()
     return _dataPaths;
 }
 
+void CrossPlatform::getTime(TimeInfo * out) {
+    if (!out)
+        throw Exception("nullptr passed into getTime");
+#if defined(__MACH__)
+    static mach_timebase_info_data_t timebase;
+    mach_timebase_info(&timebase);
```

Comment:
Static variable will be filled every time.
Do you expect that? If yes, then why do you use `static` ?